docs/reference/user-variables.rst

```diff
@@ -345,6 +345,17 @@ To create new URLs the following template is used:
 
     This is the same structure as Hunter uses for its own :ref:`Download <layout deployed download>` directory.
 
+.. note::
+
+    ``HUNTER_DOWNLOAD_SERVER`` will be applied only to packages enabled with the standard 
+    ``VERSION`` variant of ``hunter_config()`` entries, which is the case for all default
+    Hunter package definitions.  Custom package definitions introduced with a `URL/SHA1``
+    variant on ``hunter_config()`` in a project's local configuration, such as those included through 
+    ``FILEPATH`` or ``LOCAL`` arguments to ``HunterGate()``, will be unaffected by this variable,
+    but can be similarly transformed through explicit calls to ``hunter_download_server_url()``.  
```

Comment:
`hunter_download_server_url` is an internal module at this moment. I think if we need this functionality, then it's better to upgrade `hunter_config`.